contributing.md

```diff
@@ -13,7 +13,7 @@ Everyone is welcome! People of all experience levels can join, begin contributin
 Feel free to ask questions by opening an issue on the relevant library. Maintainers are also active on:
 
 - The [PureScript Discourse](https://discourse.purescript.org) (the most popular option and best for detailed questions)
-- The [Functional Programming Slack](https://functionalprogramming.slack.com) ([link to join](https://fpchat-invite.herokuapp.com)!) in the `#purescript` and `#purescript-beginners` channels (best for quick, informal questions)
+- The [Purescript Discord](https://purescript.org/chat) in the `#help-language`, `#help-code` and `#help-tools` channels (best for quick, informal questions)
```

Comment:
I’d prefer we avoid naming specific channels in case they change over time (which they have already!) but otherwise 👍

Reply:
good point, I still think its better then it was, maybe I could have fixed it with a new PR, to keep up the flow :).